src/components/iconNetflex/index.tsx

```diff
@@ -0,0 +1,13 @@
+import Fds from "../../assets/NetflixLogoSvg.svg";
```

Comment:
O nome desse import não está muito significativo.
```suggestion
import Logo from "../../assets/NetflixLogoSvg.svg";
```